pkg/controller/pod_volume_backup_controller.go

```diff
@@ -234,6 +234,11 @@ func (c *podVolumeBackupController) processBackup(req *velerov1api.PodVolumeBack
 			return c.fail(req, errors.Wrap(err, "error setting restic cmd env").Error(), log)
 		}
 		resticCmd.Env = env
+	} else if strings.HasPrefix(req.Spec.RepoIdentifier, "s3") {
```

Comment:
let's reword the comment on L230 to generically describe that we're getting provider-specific env vars to use for the restic command, since this code section is not just for Azure anymore.